cqf-fhir-cr/src/test/resources/org/opencds/cqf/fhir/cr/plandefinition/dstu3/child-routine-visit/child_routine_visit_careplan.json

```diff
@@ -5,11 +5,6 @@
     {
       "resourceType": "RequestGroup",
       "id": "ChildRoutineVisit-PlanDefinition-1.0.0",
-      "meta": {
```

Comment:
Question: What was the impetus for removing the `profile`? Just the fact that declaring your own profile is an anti-pattern in FHIR?

Reply:
Given that  a resource can adhere to any number of profiles, it was decided that the calling source should be responsible for adding in and validating expected profiles.